### PR TITLE
implement getSubscription

### LINKED ISSUE #421: implement getSubscription
https://developers.facebook.com/docs/graph-api/webhooks/subscriptions-edge

### PR DESCRIPTION
**Pull request by wtflink**
close #421 

the sample response:
```js
{
  "data": [
    {
      "object": "user",
      "callback_url": "https://your-clever-domain-name.com/webhooks",
      "active": true,
      "fields": [
        {
          "name": "photos",
          "version": "v2.10"
        },
        {
          "name": "feed",
          "version": "v2.10"
        }
      ]
    }
  ]
}
```
responded data is an array, it might have some partition problem at some point.

### CODE REVIEW

**Comment by tw0517tw** [comment]
`getSubscription` or `getSubscriptions`?

**Comment by chentsulin** [comment]
Is it possible to have two subscriptions at the same time?

**Comment by wtflink** [comment]
not quite sure yet, but I think it is possible to subscribe different object at the same time, such `user`, `page`... etc.

File: packages/messaging-api-messenger/README.md [hunk 1]
```diff
@@ -3476,6 +3476,30 @@ Default Fields:
...
+```js
+client.getSubscriptions({
+  access_token: APP_ACCESS_TOKEN,
+});
+```
+
+Or provide app id and app secret instead of app access token:
+
+```js
+client.createSubscription({
```
Comment (tw0517tw) [inline packages/messaging-api-messenger/README.md]: wrong method name

**Comment by tw0517tw** [comment]
default
It seems that we can subscribe to 4 objects, user, page, permissions and payments.

**Comment by chentsulin** [comment]
Or we can have both `getSubscriptions` and `getPageSubscription` API.

File: packages/messaging-api-messenger/README.md [hunk 2]
```diff
@@ -3476,6 +3476,54 @@ Default Fields:
 - `messaging_handovers`
 - `messaging_policy_enforcement`
 
+## `getSubscriptions`
+
+Get the current Webhook subscriptions setted up on your app.
```
Comment (chentsulin) [inline packages/messaging-api-messenger/README.md]: typo: setted

File: packages/messaging-api-messenger/README.md [hunk 3]
```diff
@@ -3476,6 +3476,54 @@ Default Fields:
...
+
+```js
+client.getSubscriptions({
+  access_token: `${APP_ID}|${APP_SECRET}`,
+});
+```
+
+## `getPageSubscription`
+
+Get the current page subscriptions setted up on your app.
```
Comment (chentsulin) [inline packages/messaging-api-messenger/README.md]: typo: subscriptions
typo: setted

**Pull request merged**